components/binary_sensor/xiaomi.py

```diff
@@ -6,67 +6,46 @@
 import logging
 import asyncio
 
-from homeassistant.helpers.entity import Entity
-from homeassistant.components.binary_sensor import (BinarySensorDevice)
-from homeassistant.const import ATTR_BATTERY_LEVEL
+from homeassistant.components.binary_sensor import BinarySensorDevice
+try:
+    from homeassistant.components.xiaomi import XiaomiDevice
+except ImportError:
+    from custom_components.xiaomi import XiaomiDevice

```

Comment:
To make sure it will work as a custom component.
Should be removed when added to HA